docs/src/main/sphinx/connector/base-jdbc.rst

```diff
@@ -0,0 +1,194 @@
+
+    # Connection URL (required for JDBC connectors)
+    connection-url=[connectorprotocol]//<host>:<port>;database=<database>
+
+    # Connection credentials
+    connection-user=USERNAME
+    connection-password=PASSWORD
+
+While the ``connector.name`` is the only required property in a catalog file,
+JDBC connectors need a specified ``connection-url``. Typically, username and
+password credentials are also required.
+
+Once the catalog properties file is established, you can add any additional
+configuration (config) properties as needed.
+
+.. _base-jdbc-config-properties:
+
+Additional config properties
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+After establishing a base catalog properties file, you can add other
+configuration properties. Connector-specific properties are defined in the
+documentation for each connector.
+
+.. list-table:: Available config properties
+  :widths: 30, 40, 30
+  :header-rows: 1
+
+  * - Property name
+    - Description
+    - Default value
+  * - ``case-insensitive-name-matching``
+    - Supports case insensitive database and collection names
+    - False
+  * - ``case-insensitive-name-matching.cache-ttl``
+    -
+    - 1 minute
+  * - ``jdbc-types-mapped-to-varchar``
+    - Allow forced mapping of comma separated lists of data types to convert to
+      unbounded ``VARCHAR``
+    -
+  * - ``metadata.cache-ttl``
+    -  Duration for which meta data, including table and column statistics, is
+       cached
+    - 0 - disabled caching
+  * - ``metadata.cache-missing``
+    - Cache the fact that meta data, including table and column statistics, is
+      not available
+    - False
+
+.. _base-jdbc-performance:
+
+Performance
+-----------
+
+The JDBC connector base includes a number of features to improve connector
+performance. These features need to be extended in the connector to be available
+as user facing features.
+
+.. _base-jdbc-table-statistics:
+
+Table statistics
+^^^^^^^^^^^^^^^^
+
+Connectors supports :doc:`table and column statistics </optimizer/statistics>`.
+They are used for :doc:`cost based optimizations
+</optimizer/cost-based-optimizations>` to improve query processing performance
+based on the actual data in the data source.
+
+Collection of statistics varies for each data source.
+
+Table and column statistics are enabled by default. You can disable it for a
+catalog by setting ``statistics.enabled`` to ``false``. The ``metadata``
+properties from the preceding section further control the statistics caching.
+
+The statistics can be viewed in Trino using :doc:`/sql/show-stats`.
+
+.. _base-jdbc-join-pushdown:
+
+Join pushdown
+^^^^^^^^^^^^^
+
+Join pushdown allows the connector delegate the table join operation to the
+underlying data source. This can result in performance gains and allows Trino to
+perform the remaining query processing on a smaller amount of data.
+
+The specifics for the supported pushdown of table joins varies for each data
+source and therefore for each connector.
```

Comment:
I think it's nice to have "Join pushdown" explained, as in a vocabulary.
I think we could have a dedicated vocabulary page. This could alleviate the need (and the temptation) to document base-jdbc, which isn't something user need to be aware of, while avoiding need to explain same term over and over again on each connector's doc page.

Reply:
Concepts page is not a bad short-term option, if we could get all such explanation snippets moved there. That's quite a job in itself to identify and move such.

Reply:
There is a concepts page already .. but is is high level only. I think it actually should be explained in the optimizer docs ... but that would all fall into as separate documentation effort since whatever we document there should be generic for all connectors and have a more complete list.. we can look at this in a follow up PR.